<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowAllcars</class>
 <widget class="QDialog" name="ShowAllcars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="L_ShowAllCarsMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Всички автомобили </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Back">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>625</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_NextRepair">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>640</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/NextRepair_Inactive.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_PrevRepair">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>640</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/PrevRepair_Inactive.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="AutoData">
    <item row="1" column="0">
     <widget class="QLabel" name="L_Repair_Marka">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Марка</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="LText_RepairYear"/>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLineEdit" name="LText_RepairVIN"/>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="LText_RepairModel"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="L_Repair_RegNumber">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Регистрационен 
 номер</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="L_Repair_Year">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Година</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="L_Repair_Fuel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Гориво</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="L_Repair_Type">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Купе</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="LText_RepairFuel"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="L_Repair_Model">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Модел</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="LText_RepairType"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="L_Repair_VIN">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>РАМА / VIN /</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="LText_RepairMarka"/>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="LText_RepairRegNumber"/>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="L_RepairAutoInfo">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Данни за автомобила </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="L_NextRepair">
   <property name="geometry">
    <rect>
     <x>756</x>
     <y>593</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Следващ 
 ремонт</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L_PrevRepair">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>590</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Предишен 
 ремонт</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="L_RepairDescr">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Описание на ремонта</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Search">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ТЪРСИ</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="SearchOptions">
    <item row="0" column="0">
     <widget class="QRadioButton" name="RButton_SearchAutos">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Търсене  по 
 автомобил</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="RButton_SearchClients">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Търсене по 
 клиент</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button_FromDate">
      <property name="text">
       <string>От дата</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button_ToDate">
      <property name="text">
       <string>До дата</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="RButton_SearchRegNumbers">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Търсене по 
 Рег. Номер</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="LText_SearchTOdate"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="LText_SearchFROMdate"/>
    </item>
    <item row="3" column="0" rowspan="2">
     <widget class="QRadioButton" name="RButton_SearchDate">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Търсене 
 по дата</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QComboBox" name="Combo_SearchAuto">
      <item>
       <property name="text">
        <string>Автомобил</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QComboBox" name="Combo_Search_Klient">
      <item>
       <property name="text">
        <string>Клиент</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QComboBox" name="Combo_Search_RegNumber">
      <item>
       <property name="text">
        <string>Рег. Номер</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="RepairsList">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>91</y>
     <width>721</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
